web_src/js/features/repo-common.js

```diff
@@ -96,6 +100,66 @@ export function initRepoCloneLink() {
   });
 }
 
+const initInputCitationValue = () => {
+  const $citationCopyApa = $('#citation-copy-apa');
+  const $citationCopyBibtex = $('#citation-copy-bibtex');
+  const {citiationFileContent} = pageData;
+  const config = plugins.config.get('@bibtex');
+  config.constants.fieldTypes.doi = ['field', 'literal'];
+  config.constants.fieldTypes.version = ['field', 'literal'];
+  const citationFormatter = new Cite(citiationFileContent);
+  const apaOutput = citationFormatter.format('bibliography', {
+    template: 'apa',
+    lang: 'en-US'
+  });
+  const bibtexOutput = citationFormatter.format('bibtex', {
+    lang: 'en-US'
+  });
+  $citationCopyBibtex.attr('data-text', bibtexOutput);
+  $citationCopyApa.attr('data-text', apaOutput);
+};
```

Comment:
Sorry, what I meant with my comment is that you should remove the function around it and leave the code at the same place(as it's only called once).

Reply:
Ah I see, thanks for the clarification. On hindsight it isn't clear that one is preparing the values and the other is interacting with the user's events. See my other comment of why the tests are failing.